torch/_dynamo/variables/builder.py

```diff
@@ -419,6 +419,21 @@ def index_source(key):
                 source=self.source,
                 guards=make_guards(GuardBuilder.BUILTIN_MATCH),
             )
+        # Rather annoying, if it takes a processgroup, it needs to go here to
+        # force inlining
+        elif value in [
```

Comment:
@voznesenskym is this the part that you were asking for help with?  is it broken currently or its just annoying?